content/docs/concepts/accountability/_index.md

```diff
@@ -0,0 +1,380 @@
+A direct fault proof can be reported at any time.
+{{% /alert %}}
+
+Slashing for _faults_ is applied at the end of each epoch. For each validator with _fault(s)_ slashing is applied for the _fault_ with the highest _severity_ (grouping faults by epoch).
+
+After successful [handling and verification](/reference/api/aut/op-prot/#handleevent-accountability-contract) of a directly submitted _fault_ on-chain a `NewFaultProof` event is emitted logging the _offending_ [validator identifier](/concepts/validator/#validator-identifier) address, _severity_ of rule infraction, and the event ID.
+
+#### Slashing and severity
+
+The AFD protocol will only apply slashing to an _offending_ validator once for epoch, only slashing the reported _fault_ with the highest _severity_ in an epoch. Note that a validator can still be slashed for multiple offenses at the end of an epoch, if the sanctioned faults were committed in past distinct epochs.
+
+Rule infraction _severity_ has two key influences:
+
+- determining if a new _fault_ is created or not, conditional on the _offending_ validator not having an existing _fault_ with a _severity_ `>=` to that of the candidate new _fault_
+- determining the amount of the slashing applied.
+
+Slashing amount is calculated by a number of parameters, including _severity_. 
+
+For the severity taxonomy see [Rule severity](/concepts/accountability/#rule-severity).
+
+For slashing calculation and parameters see [Slashing](/concepts/accountability/#slashing).
+
+{{% alert title="Note" %}}
+A validator can be slashed for committing a fault in an epoch more than once in an edge case scenario. For example:
+
+- **Epoch `n`**: _offending_ validator slashed for fault committed in epoch `n`
+- **Epoch `n+1`**: new _faults_ are reported for the _offending_ validator: (1) one for the current **epoch `n+1`**; (2) one for **epoch `n`** with a higher _severity_ than the _fault_ already slashed in **epoch `n`**.
+- The _offending_ validator has two slashings applied applied in **epoch `n+1`**:
+  - for epoch `n+1`
+  - for epoch `n`.
+{{% /alert %}}
+
+
+#### Jail
+
+Jailing is a protocol action that excludes a validator from selection to the consensus committee for a period of time measured as a number of blocks.
+
+Jailing may be applied as part of a slashing penalty depending on the _severity_ of the _fault_ being _slashed_. Jailing changes a validator's state from `active` to `jailed`. Validators in a `jailed` state are debarred from [consensus committee selection](/concepts/consensus/committee/#committee-member-selection).
+
+The jail period is computed by `current block number + jail factor * proven fault count * epoch period` where:
+
+- `current block number`: the block number at the time of computation (i.e. the last block of an epoch when slashing is applied)
+- `jail factor`: a set number of epochs, defined as a protocol parameter in the [slashing protocol configuration](/concepts/accountability/#slashing-protocol-configuration).
```

Comment:
I've removed the period and reworded in commit 729649df83fefb59e35069f061a1f11f0045328f.

Reply:
This doesn't get at the meaning, context, or relevance.

Imagine defining "volume" as "something measured in liters"... Instead of saying "volume is the amount of space that something occupies, which can be measured in..." It's the former part that is missing.

Reply:
Yes the _jail factor_ definition is improved